sharktank/sharktank/utils/tokenizer.py

```diff
@@ -76,9 +91,10 @@ class _TransformersTokenizer(InferenceTokenizer):
         def __init__(self, t: AutoTokenizer):
             self._t = t
 
-        def _encode(self, texts: list[str]) -> list[list[int]]:
+        def _encode(self, texts: list[str], add_start_token: bool) -> list[list[int]]:
             results = t.batch_encode_plus(
                 texts,
+                add_special_tokens=add_start_token,
```

Comment:
I was struggling to find this param. Thanks!

Reply:
Yes, here I am trying to not add a start token, but by default it's enabled. Couple of other variations, llama.cpp and HF, also pad on the left rather than the right (like we do) for their perplexity evaluation. Bunch of knobs to tune for sure.